IceCubesApp/App/Tabs/Settings/SettingsTab.swift

```diff
@@ -82,6 +84,9 @@ struct SettingsTabs: View {
       NavigationLink(destination: remoteLocalTimelinesView) {
         Label("Remote Local Timelines", systemImage: "dot.radiowaves.right")
       }
+      Toggle(isOn: $preferences.useInAppSafari) {
```

Comment:
Could you make it a Picker instead? It'll make the label shorter and also more clear like "Browser: In App Safari, System Safari" as choices.

Reply:
On a small iPhone simulator, the label does not fit on one line.

Reply:
@Dimillian done, let me know if the implementation of it is ok.